src/analyzer/protocol/ssl/SSL.h

```diff
@@ -45,10 +49,123 @@ class SSL_Analyzer final : public analyzer::tcp::TCP_ApplicationAnalyzer
 
 	static analyzer::Analyzer* Instantiate(Connection* conn) { return new SSL_Analyzer(conn); }
 
+	/**
+	 * Set the secret that should be used to derive keys for the
+	 * connection. (For TLS 1.2 this is the pre-master secret)
+	 *
+	 * Please note that these functions currently are hardcoded to only work with a single TLS 1.2
+	 * cuphersuite (TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384).
```

Comment:
```suggestion
	 * ciphersuite (TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384).
```
More of these below.